src/components/NewPassword.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import { resetPassword } from "../Firebase/UserUtils";
+import { successToast, errorToast } from "../utils/toast";
+import PrimaryButton from "../components/PrimaryButton";
+import "../css/NewPassword.scss";
+
+class NewPassword extends Component {
+    state = {
+      email: "",
+    };
+
+    handleChange = (event) => {
+      if (event.target.type === "email") {
+        this.setState({ email: event.target.value });
+      }
+    }
+
+    retrievePasswordClick = (event) => {
+      event.preventDefault();
+      const { email } = this.state;
+      resetPassword(email)
+        .then(() => {
+          successToast(`An email to ${email} has been sent`);
+        }).catch((error) => {
+          errorToast(error.message);
+        });
+    }
+
+    render() {
+      return (
+        <div id="new-password" className="container">
+          <h3>Password Reset</h3>
```

Comment:
Use `h1` instead, since this isn't a "subsection" of anything. You can make the font smaller in CSS